edit-readme-to-create-tables

q1umi-ooki opened the pull request:
# WHAT
## I planned structure of four tables. 
## users table, groups table, members table, and chats table

# WHY
## Because, I want to create DATABASE to save these data correctly.

Commit:
edit README.md to create TABLES

I planned about structures of tables.
I added 4 tables at README.md
users table, groups table, members table, and chats table.

Commit:
Edit README.md to create tables

I planned about structures of tables.
I added 4 tables at README.md
users table, groups table, members table, and chats table.

kuroko918 commented on README.md:
```diff
@@ -22,3 +22,64 @@ Things you may want to cover:
 * Deployment instructions
 
 * ...
```
この行より上の不要な記述は消してしまいましょう！

kuroko918 commented on README.md:
```diff
@@ -22,3 +22,64 @@ Things you may want to cover:
+
+### Association
+- has_many :users
+- has_many :groups
```
ここ２行はuser_id、group_idと外部キーを持つのにhas_manyを持つことはありません。
もう一度、user及びgruopとmessageの関係について考えてみましょう！

kuroko918 commented on README.md:
```diff
@@ -22,3 +22,64 @@ Things you may want to cover:
+
+|Column|Type|Options|
+|------|----|-------|
+|group_name|string|null: false, unique: true|
```
こちらもカラム名はnameで大丈夫です。userのnameと混同することはないかと思います。

kuroko918 commented on README.md:
```diff
@@ -22,3 +22,64 @@ Things you may want to cover:
+
+***
+
+## 4.chatsテーブル
```
こちらは機能的にまっったく問題はないのですが、カリキュラムがmessagesテーブルとなっているので、messagesテーブルとするとカリキュラムが読みやすいかもしれません。別に間違えではないので、お好きな方を選択ください。

Commit:
Edit-readme-to-create-database

I planned about structures of tables.
I added 4 tables at README.md
users table, groups table, members table, and chats table.
I fixed 4 part

Commit:
Edit-readme-to-create-tables

I planned about structures of tables.
I added 4 tables at README.md
users table, groups table, members table, and chats table.
I fixed 5 section.

q1umi-ooki commented:
早速のレビューありがとうございます。
4箇所修正しました。
・22行目以上の不要部分の削除#READMEを除いた全てを削除。
・members テーブル　has_many部分　belongs_toへ修正してます。1対1の関係でしたね。
・groups テーブル　group_name →　nameへ修正
・chatsテーブル　chats → messages へカリキュラムに合わせて修正

加えて、messagesテーブルの
referencesの綴りミスを発見してしまったので修正しております。
ご確認お願いいたします。

TOMARUMARU reviewed (approved):
LGTM

q1umi-ooki commented:
ありがとうございます！
